### DISCUSSION TITLE
鉴于杜比大喇叭的作者已经不再活跃，想问下在安卓上是否还有其他的可用方式？

### DISCUSSION THREAD

**Discussion by ssrubsd** [1715173329/UnblockNeteaseMusic#1142] [General]
### 需求描述

或者说官方有考虑直接支持安卓端吗

**Comment by yujincheng08** [comment]
大喇叭如果我使用时候遇到问题，我会提交修复的

**Comment by 1715173329** [comment]
也可以用 termux 运行核心然后设置代理，或者使用我们的 OpenWrt 插件：
https://github.com/UnblockNeteaseMusic/luci-app-unblockneteasemusic

**Comment by chris7395328** [comment]
我还是继续用大喇叭，服务器自定义就可以了，我是部署在群晖docker正常用
